cointoss/app/views/rooms/show.html.erb

```diff
@@ -1,39 +1,76 @@
 <% user = User.find_by(id: @room.host_id) %>
 <% username = "#ERR" %>
 <% username = user.username if user %>
-<h1> <%= @room.name %> hosted by <%= username%></h1>
-<% if current_user.id == @room.host_id %>
-    <%= form_for @room do |r|%>
-        <%= fields_for :actions do |a|%>
-            <%= a.label "Action Description"%><br>
-            <%= a.text_field :description%><br>
-            <%= a.label "Percent Success"%><br>
-            <%= a.number_field :odds%><br>
-            <%= a.hidden_field :room_id, value: @room.id%>
-            <%= a.submit "Add Action"%>
+<h1>Welcome to <%= @room.name %>, hosted by <%= username %></h1>
+
+<% if @room.users.exists?(current_user.id)%>
+    <%= button_to "Leave", {:controller => :rooms, :action => 'remove_user', :user_id => current_user.id, :room_id => @room.id}, method: :post%>
+<% end %>
+
+<%= form_with(url: room_actions_path(@room), method: :post) do |r| %>
+    <% if current_user.id == @room.host_id %>
+        <%= fields_for :actions do |a| %>
+            <%= a.label "Action Description" %><br>
+            <%= a.text_field :description %><br>
+            <%= a.label "Percent Success" %><br>
+            <%= a.number_field :odds %><br>
+            <%= a.hidden_field :room_id, value: @room.id %>
+            <%= a.submit "Add Action" %>
         <% end %>
     <% end %>
-<% elsif @room.users.exists?(current_user.id)%>    
-    <%= button_to "Leave", {:controller => :rooms, :action => 'remove_user', :user_id => current_user.id, :room_id => @room.id}, method: :post%>
 <% end %>
-<table>
-    <tr>
-        <th style="text-align:center" class="small"  >ID</th>
-        <th style="text-align:center" class="big">Action Description</th>
-        <th style="text-align:center" class="small">Odds</th>
-        <% if current_user.id == @room.host_id %>
+
+<% if current_user.id == @room.host_id %>
+    <table>
+        <tr>
+            <th style="text-align:center" class="small">ID</th>
+            <th style="text-align:center" class="big">Action Description</th>
+            <th style="text-align:center" class="small">Odds</th>
             <th style="text-align:center" class="small">Options</th>
+        </tr>
+
+        <% @room.actions.each do |action| %>
+            <tr>
+                <td style="text-align:left" class="small"><%= action.id %></td>
+                <td style="text-align:left" class="big"><%= action.description %></td>
+                <td style="text-align:left" class="small"><%= action.odds %></td>
+
+                <td style="text-align:left" class="small"><%= button_to "Begone Thot", {controller: :actions, action: :destroy, room_id: @room.id, id: action.id}, method: :delete %></td>
```

Comment:
"Begone Thot"?? Really?? I expect a certain level of professionalism from my employees and this is not it.